lib/src/kbasesearchengine/main/DecorateWithNarrativeInfo.java

```diff
@@ -0,0 +1,121 @@
+package kbasesearchengine.main;
+
+import kbasesearchengine.*;
+import kbasesearchengine.ObjectData;
+import kbasesearchengine.common.GUID;
+import us.kbase.common.service.JsonClientException;
+import kbasesearchengine.events.handler.CloneableWorkspaceClientImpl;
+import kbasesearchengine.events.handler.EventHandler;
+import kbasesearchengine.events.handler.WorkspaceEventHandler;
+import us.kbase.common.service.Tuple5;
+import us.kbase.common.service.Tuple9;
+import workspace.*;
```

Comment:
same

Reply:
Done. (It was there in the original code.)